microsoft-edge/devtools-guide-chromium/overview.md

```diff
@@ -259,23 +324,41 @@ The main toolbar contains the following features:
    *  **Performance** tool.
    *  **Memory** tool.
    *  **Application** tool.
-   *  **Security** tool.
-   *  **Lighthouse** tool.
-   *  **CSS Overview** tool.
 
 * Icons:
-   *  **More tabs** (![More tabs icon](./overview-images/more-tabs-icon-light-theme.png)) button.
+   *  **Move Activity Bar to left** (![Move Activity Bar to left icon](./overview-images/move-activity-bar-to-left-icon.png)) button.
    *  **More Tools** (![More Tools icon](./overview-images/more-tools-icon-light-theme.png)) button.
-   *  **JavaScript errors counter** (![JavaScript errors counter icon](./overview-images/javascript-counter-icon-light-theme.png)) button.
-   *  **Issues counter** (![Issues counter icon](./overview-images/issues-counter-icon-light-theme.png)) button.
-   *  **Settings** (![Settings icon](./overview-images/settings-gear-icon-light-theme.png)) button.
-   *  **Send Feedback** (![Send Feedback icon](./overview-images/send-feedback-icon-light-theme.png)) button.
    *  **Customize and control DevTools** (![Customize icon](./overview-images/customize-devtools-icon-light-theme.png)) menu button.
+   *  **Help** (![Help icon](./overview-images/help-icon.png)) button.
    *  **Close** DevTools (![Close DevTools icon](./overview-images/close-devtools-icon-light-theme.png)) button.
 
 ![The main toolbar, with labels that identify its features](./overview-images/devtools-intro-menu-bar.png)
 
-The toolbar features are described below.
+The **Activity Bar** features are described below.
+
+
+<!-- ------------------------------ -->
+#### Pin and rearrange tools in the Activity Bar
+
+You can choose which tools appear in the **Activity Bar** by pinning or unpinning tools. This allows you to customize DevTools to your preferred workflow.
+
+Click **More tools** (**+**) to list all the available tools. Selecting a tool will pin it to the **Activity Bar**. It will be shown by default every time DevTools is opened.
```

Comment:
```suggestion
Click **More tools** (**+**) to list all the available tools. Selecting a tool will pin it to the **Activity Bar**. The tool is then present in the Activity Bar whenever DevTools is opened.
```
* "it" - i avoid indirection pointer, unless that makes same word appear twice in a row.
* "by default" - might add unnecessary complexity; the main idea is that the tool stays there.